tensorboardX/summary.py

```diff
@@ -369,25 +368,46 @@ def video(tag, tensor, fps=4, dataformats="NTCHW"):
 
 def make_video(tensor, fps):
     try:
-        import moviepy  # noqa: F401
+        import moviepy
     except ImportError:
         print('add_video needs package moviepy')
         return
     try:
-        from moviepy import editor as mpy
+        # moviepy v2+
+        from moviepy import ImageSequenceClip
     except ImportError:
-        print("moviepy is installed, but can't import moviepy.editor.",
-              "Some packages could be missing [imageio, requests]")
-        return
+        try:
+            # Fallback for all moviepy versions
+            from moviepy.video.io.ImageSequenceClip import ImageSequenceClip
+        except ModuleNotFoundError as e:
```

Comment:
How about combine the two excepts into `except (ImportError, ModuleNotFoundError) as e:`?